README.md

```diff
@@ -2,6 +2,48 @@
 
 An open-source implementation of the [The United States Web Design System ](https://designsystem.digital.gov/) (USWDS) version 2.11.1 using the [Hugo](https://gohugo.io/) open-source static site generator.
 
-Implementation of the USWDS is currently incomplete, but suffecient to support a number of websites hosted at NIST. [Contributions](CONTRIBUTING.md) to improve this project are welcome.
+Implementation of the USWDS is currently incomplete, but sufficient to support a number of websites hosted at NIST. [Contributions](CONTRIBUTING.md) to improve this project are welcome.
 
 This project has been tested on HUGO version >= [0.58.3](https://github.com/gohugoio/hugo/releases/latest) and requires a `hugo-extended` build with pipe and SCSS generation support. Installation [instructions](https://gohugo.io/getting-started/installing) for common platforms are available in the Hugo [documentation](https://gohugo.io/documentation/).
+
+## Usage
+
+For details on using this theme, refer to [the documentation and demo site](https://pages.nist.gov/hugo-uswds-docs/).
+
+### Installation
+
+This template currently supports installation via Hugo Modules or as a Git Submodule.
+
+#### Using Hugo Modules
+
+For detailed instruction, refer to the [Hugo Module documentation](https://gohugo.io/hugo-modules/).
+
+1. From you project directory, initialize Hugo Modules:
+
+    ```
+    $ hugo mod init $REPOSITORY
+    ```
+
+    *NOTE: here `$REPOSITORY` should be the path to your repository (e.g. `github.com/<username>/<repository>`)*
```

Comment:
Is this the theme repository or the website repository?

Reply:
```suggestion
    *NOTE: here `$REPOSITORY` should be the path to your website repository (e.g. `github.com/<username/organization>/<repository>`)*
```